File: PHP PowerPoint Examples/TableApiDemo/getSlideTable.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\TableApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TableApi($configuration);

$name = "SlideTable.pptx"; 
$password = null; 
$folder = "Powerpoint"; 
$storage = null; 
$slideIndex = 0;  // int | slide index
$shapeIndex = "0"; // string | Shape index.
$result = $apiInstance->getSlideTable($name,$slideIndex,$shapeIndex, $password, $folder, $storage);

File: Python PowerPoint Examples/TableApiDemo/getSlideTable.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.table_api.TableApi(configuration)

fileName = "SlideTable.pptx"
password = ""
storage = ""
slideIndex = 0
shape_index = 0
inputFolder = "input"
result = api.get_slide_table(fileName, slide_index=slideIndex, shape_index=shape_index, password=password, storage=storage,folder=inputFolder)